extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -325,13 +325,6 @@ export async function defaultAzureAccountServiceFactory(): Promise<vscodeMssql.I
 export async function getDefaultPublishDeploymentOptions(project: ISqlProject): Promise<mssql.DeploymentOptions | vscodeMssql.DeploymentOptions> {
 	const schemaCompareService = await getSchemaCompareService();
 	const result = await schemaCompareService.schemaCompareGetDefaultOptions();
-	// re-include database-scoped credentials
-	if (getAzdataApi()) {
-		result.defaultDeploymentOptions.excludeObjectTypes.value = (result.defaultDeploymentOptions as mssql.DeploymentOptions).excludeObjectTypes.value?.filter(x => x !== mssql.SchemaObjectType.DatabaseScopedCredentials);
-	} else {
-		result.defaultDeploymentOptions.excludeObjectTypes.value = (result.defaultDeploymentOptions as vscodeMssql.DeploymentOptions).excludeObjectTypes.value?.filter(x => x !== vscodeMssql.SchemaObjectType.DatabaseScopedCredentials);
-	}
-
```

Comment:
Why does it no longer need to re-included?  Is this done down in STS?

Reply:
since we had no deployment options in SQL Proj before but deploymentoptions.excludeObjectTypes had some (around 22) default objects excluded for SC. We were reincluding the DatabaseScopedCredentials from the excludeObject types for publish. But now we have options functionality which takes care of these values, we do not need this anymore. Thanks